tests/performance/test_performance.py

```diff
@@ -76,6 +77,9 @@ def _get_performance_test_cases():
         dataset_generators_for_type = dataset_generators.get(data_type, [])
 
         for transformer in transformers_for_type:
+            if transformer == BayesGMMTransformer:
+                continue
```

Comment:
Talking to @kveerama , I think it might be a good idea to create a list of `Sandbox` transformers. These by definition will be transformers that have room for improvement in the performance area. Could you make that list and add this transformer to it?